src/families/ethereum/EthereumCustomFees.js

```diff
@@ -0,0 +1,150 @@
+/* @flow */
+import React, { useState, useCallback } from "react";
+import { StyleSheet, View } from "react-native";
+import { BigNumber } from "bignumber.js";
+import invariant from "invariant";
+import { useSelector } from "react-redux";
+import { useTheme } from "@react-navigation/native";
+import { inferDynamicRange } from "@ledgerhq/live-common/lib/range";
+import { getAccountBridge } from "@ledgerhq/live-common/lib/bridge";
+import { i18n } from "../../context/Locale";
+import { accountScreenSelector } from "../../reducers/accounts";
+import EditFeeUnitEthereum from "./EditFeeUnitEthereum";
+import SectionSeparator from "../../components/SectionSeparator";
+import Button from "../../components/Button";
+import EthereumGasLimit from "./SendRowGasLimit";
+
+import type { RouteParams } from "../../screens/SendFunds/04-Summary";
+
+type Props = {
+  navigation: any,
+  route: { params: RouteParams },
+};
+
+const options = {
+  title: i18n.t("send.summary.fees"),
+  headerLeft: null,
+};
+
+const fallbackGasPrice = inferDynamicRange(BigNumber(10e9));
+let lastNetworkGasPrice; // local cache of last value to prevent extra blinks
```

Comment:
isnt that gas price logic present on live-common already? :thinking:

Reply:
I reuse the code from what it already exist, not sure if it's the right way or logic, so not so sure how to answer that :/

Reply:
i though we had those values somewhere on live-common already my bad.